client/extensions/woocommerce/state/sites/shipping-methods/selectors.js

```diff
@@ -3,24 +3,50 @@
 /**
  * External dependencies
  */
-
-import { find, get, isArray } from 'lodash';
+import { translate } from 'i18n-calypso';
+import { filter, get, isArray } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import createSelector from 'lib/create-selector';
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { isWcsEnabled } from 'woocommerce/state/selectors/plugins';
 import { LOADING } from 'woocommerce/state/constants';
 
+/*
+ * By default, those methods are called "XXXX (WooCommerce Services)".
+ * We aren't pushing the "WooCommerce Services" brand anywhere in Calypso, so the method names must be changed.
+ */
+const METHOD_NAMES = {
+	wc_services_usps: translate( 'USPS' ),
```

Comment:
Probably need to add `comment` here for translators for "United States Postal Service"